File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        String strA = Integer.toString(a);
        String strB = Integer.toString(b);

        String sumA = strA + strB;
        String sumB = strB + strA;

        return Math.max(Integer.parseInt(sumA), Integer.parseInt(sumB));
    }
}

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.kt
class Solution {
    fun solution(a: Int, b: Int): Int {
        val strA: String = a.toString()
        val strB: String = b.toString()
        val sumA: String = strA + strB
        val sumB: String = strB + strA
        return sumA.toInt().coerceAtLeast(sumB.toInt())
    }
}